charts/galoy/templates/api-ingress.yaml

```diff
@@ -15,6 +15,9 @@ metadata:
 
     nginx.ingress.kubernetes.io/proxy-read-timeout: "3600" # 1 hour
     nginx.ingress.kubernetes.io/proxy-send-timeout: "3600" # 1 hour
+    nginx.ingress.kubernetes.io/proxy-connect-timeout: "5s"
+    nginx.ingress.kubernetes.io/proxy-next-upstream: "error timeout"
+    nginx.ingress.kubernetes.io/proxy-next-upstream-tries: "3"
```

Comment:
by default there is no retry?

Reply:
i think it only does if you use `proxy_pass`
<img width="759" alt="Screen Shot 2023-04-18 at 2 32 09 PM" src="https://user-images.githubusercontent.com/1273575/232885602-aa29c589-9ccf-4f5d-8873-3a3f49b43878.png">